website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,173 @@
+---
+title: '0.72 - Beta Support for Package Exports and Symlinks'
+authors: [kelset, fortmarek, lunaleaps, cipolleschi]
+tags: [announcement, release]
+date: 2023-05-30
+---
+
+Today we’re releasing 0.72!
+
+This release adds a couple of highly requested features for Metro and developer experience improvements. So much of this work stems from your feedback on the [2022 community survey](https://github.com/react-native-community/discussions-and-proposals/discussions/528) -- thank you to all those that participated!
+
+### Highlights
+
+- [Metro support for Package Exports, symlinks, and config changes](/blog/2023/05/30/0.72-metro-package-exports-symlinks#metro)
+- [Developer experience improvements](/blog/2023/05/30/0.72-metro-package-exports-symlinks#developer-experience-improvements)
+- [Moving New Architecture updates](/blog/2023/05/30/0.72-metro-package-exports-symlinks#react-native-package-renames)
+- [Mitigation for TextInput ANRs on Android](/blog/2023/05/30/0.72-metro-package-exports-symlinks#react-native-package-renames)
+
+### Breaking Changes
+
+- [Deprecated Component Removals](/blog/2023/05/30/0.72-metro-package-exports-symlinks#deprecated-component-removals)
+- [Package renames](/blog/2023/05/30/0.72-metro-package-exports-symlinks#package-renames)
+
```

Comment:
Is this relevant to a user who is not actively using New Architecture?

Reply:
Technically not, but for library maintainers is a problem. I was talking also with @kelset and we think it would be a good thing to call this out in the breaking changes as this is probably the only breaking changes for the New Arch in 0.72 (0.73 will be a completely different story)